docs/pages/includes/cli-completion.mdx

```diff
@@ -0,0 +1,28 @@
+<Details title="Improve the CLI experience: enable shell completion">
+
+  Out of the box, CLI tools can take advantage of completion hints for flags and commands.
+  For example, typing `tsh` and pressing `Tab` will show all available subcommands, typing `tsh --` and pressing `Tab`
+  will show all available flags.
+
+  Supported shells are bash and zsh.
+
+  To enable completion add an additional statement to `bash_profile` (or equivalent).
+  Replace `<cli-tool>` with `teleport`, `tsh`, `tctl` or `tbot`.
```

Comment:
I'm not sure if `<` `>` are needed (do we have a convention for that?).

Reply:
I would just be specific (use `tsh` as the example), and then add a sentence at the very end that you can repeat the same process for `tctl`, `teleport`, and `tsh`.

(In practice, most people will only use this for tsh and tctl, teleport and tbot are typically run as daemons and not often invoked manually.